SocketRocket.xcodeproj/project.xcworkspace/xcshareddata/SocketRocket.xccheckout

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be in our .gitignore. We're adding that now. Please remove this file from your PR.